fdbclient/ServerKnobs.cpp

```diff
@@ -854,6 +854,10 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
     init( ENCRYPTION_MODE,                              "AES-256-CTR");
     init( SIM_KMS_MAX_KEYS,                                      4096);
 
+    // Support KmsConnector types are:
+    // KMS_CONNECTOR_TYPE_HTTP -> 1
+    init( KMS_CONNECTOR_TYPE,                      "HttpKmsConnector");
```

Comment:
EKP recruitment is gated by the ENABLE_ENCRYPTION knob, right?
If not we should have a "no_kms" option here and have that be the default instead of having this be default on.

Reply:
> EKP recruitment is gated by the ENABLE_ENCRYPTION knob, right?
Yes, EKP recruitment is gated by ENABLE_ENCRYPTION knob, hence, KMS connector allocation is protected by the same knob.